README.md

```diff
@@ -17,3 +17,39 @@ http://godoc.org/github.com/fhs/go-netcdf/netcdf
 How to install:
 
 	$ go get github.com/fhs/go-netcdf/netcdf
+
+## Example
+
+### Open an existing netcdf file
+
+	nc, err := netcdf.CreateFile("gopher.nc", netcdf.NOWRITE)
+	if err != nil {
+		return err
+	}
+	defer nc.Close()
+
+### Variables
+
+    var, err := nc.Var("time")
+    if err == nil {
+    	nattrs,_ := var.NAttrs()
+    	for i:=0;i<nattrs;i++ {
+    		attr, err1 := var.AttrN(i)
+    		if err1 == nill {
```

Comment:
All changes have been done, and example updated with the attribute use case. CI validation is also fine with the latest commit